packages/graph/hash_graph/lib/graph/src/store/postgres/knowledge/entity/mod.rs

```diff
@@ -122,19 +121,23 @@ impl<C: AsClient> PostgresStore<C> {
                             )
                             .await?
                             .into_iter()
-                            .map(|entity| entity.metadata().edition_id())
+                            .map(|entity| {
```

Comment:
Can you update the variable name for all of these

Reply:
Updated the names in 2507807f458d5ee53a31c04ab84d5918e93a5096 and 36c72e68e8c133d66e8e14ddb61f12bf72c38124. The names may be a little bit too long but I expect to remove these soon.

Reply:
Long names are better than unclear or incorrect ones 😁 Thanks!